extensions/wikia/AdEngine/js/provider/gpt/googleTag.js

```diff
@@ -2,58 +2,60 @@
 /*jshint maxlen:125, camelcase:false, maxdepth:7*/
 define('ext.wikia.adEngine.provider.gpt.googleTag', [
 	'ext.wikia.aRecoveryEngine.recovery.helper',
+	'ext.wikia.adEngine.adLogicPageViewCounter',
 	'wikia.document',
 	'wikia.log',
 	'wikia.window'
-], function (helper, doc, log, window) {
+], function (helper, adLogicPageViewCounter, doc, log, window) {
 	'use strict';
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Use of JavaScript strict mode may result in unexpected behaviour in some browsers. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AStrictMode)